source/_integrations/sensehat.markdown

```diff
@@ -140,8 +140,35 @@ group:
   kitchen:
     - group.sense_hat
 ```
-
-### Directions for installing on Raspberry Pi Raspbian Based installation:
+## Installation
+### On Home Assistant Operating System
+
+Things got complicated for poor _SenseHAT_ with Home Assistant Operating System. I found finally a way to use it.
```

Comment:
Your personal opinion is not something we should document.

```suggestion
Things got complicated for poor _SenseHAT_ with Home Assistant Operating System. I found finally a way to use it.
```

Reply:
removed, I thought that a bit of humor could help